File: java-play-framework/app/apimodels/RequestTimeFilterFastProperty.java
package apimodels;

import com.fasterxml.jackson.annotation.*;
import java.util.Set;
import javax.validation.*;
import com.fasterxml.jackson.annotation.JsonCreator;

/**
 * Gets or Sets RequestTimeFilterFastProperty
 */
public enum RequestTimeFilterFastProperty {
  
  TRAVEL_TIME("travel_time"),
  
  FARES("fares");

  private final String value;

  RequestTimeFilterFastProperty(String value) {
    this.value = value;
  }

  @Override
  @JsonValue
  public String toString() {
    return String.valueOf(value);
  }

  @JsonCreator
  public static RequestTimeFilterFastProperty fromValue(String value) {
    for (RequestTimeFilterFastProperty b : RequestTimeFilterFastProperty.values()) {
      if (b.value.equals(value)) {
        return b;
      }
    }
    throw new IllegalArgumentException("Unexpected value '" + value + "'");
  }
}

File: kotlin-spring/src/main/kotlin/traveltimeplatform/models/RequestTimeFilterFastProperty.kt
package traveltimeplatform.models

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonValue
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size

/**
* 
* Values: travelTime,fares
*/
enum class RequestTimeFilterFastProperty(val value: String) {

    travelTime("travel_time"),

    fares("fares");

}

